simtools/model/mirrors.py

```diff
@@ -60,27 +61,55 @@ def _read_mirror_list_from_ecsv(self):
 
         self.mirror_table = Table.read(self._mirror_list_file, format="ascii.ecsv")
         self._logger.debug(f"Reading mirror properties from {self._mirror_list_file}")
-        try:
-            self.shape_type = u.Quantity(self.mirror_table["shape_type"])[0].value
-            self.mirror_diameter = u.Quantity(self.mirror_table["mirror_diameter"])[0].value
-            self.number_of_mirrors = len(self.mirror_table["focal_length"])
 
+        self.number_of_mirrors = np.shape(self.mirror_table)[0]
+        self._logger.debug(f"Number of Mirrors = {self.number_of_mirrors}")
+
+        if self.number_of_mirrors == 0:
+            msg = "Problem reading mirror list file"
+            self._logger.error(msg)
+            raise InvalidMirrorListFile
+
+        try:
+            self.shape_type = u.Quantity(self.mirror_table["shape_type"])[0]
```

Comment:
We use `shape_type` in the tables as well. So we can use it consistent in the code.

Reply:
But does is `shape_type` clearer than just `shape`? If so, then no problem, we can resolve this conversation. Otherwise, we could change the tables as well (we have very few right now, so better decide now).